src/main/java/net/dries007/tfc/common/blocks/wood/TFCLeavesBlock.java

```diff
@@ -77,18 +77,30 @@ public static void onEntityInside(BlockState state, Level level, BlockPos pos, E
         }
     }
 
+    public static TFCLeavesBlock create(ExtendedProperties properties, int autumnIndex)
```

Comment:
I think we might be able to not use these static create methods now that the leaves blocks aren't distance sensitive anymore... try removing and using ctors?